SwedbankPaySDK/Classes/SwedbankPaySDKViewModel.swift

```diff
@@ -403,11 +406,12 @@ final class SwedbankPaySDKViewModel: NSObject {
         return problem
     }
     
-    private func getSubProblems(_ string: [Dictionary<String, Any>]?) -> [SwedbankPaySDK.SwedbankPaySubProblem]? {
-        var subProblems: [SwedbankPaySDK.SwedbankPaySubProblem]? = nil
-        if let string = string {
-            subProblems = Mapper<SwedbankPaySDK.SwedbankPaySubProblem>().mapArray(JSONObject: string)
+    private func getSubProblems(_ json: [Dictionary<String, Any>]?) -> [SwedbankPaySDK.SwedbankPaySubProblem]? {
```

Comment:
Codacy found an issue: [Shorthand syntactic sugar should be used, i.e. [String: Int] instead of Dictionary<String, Int>.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6081366)